zio-http/src/test/scala/zhttp/http/EncodeRequestSpec.scala

```diff
@@ -0,0 +1,21 @@
+package zhttp.http
+
+import io.netty.handler.codec.http.HttpVersion
+import zhttp.http.URL.Location
+import zhttp.service.EncodeRequest
+import zio.test.Assertion._
+import zio.test.{DefaultRunnableSpec, assert}
+
+object EncodeRequestSpec extends DefaultRunnableSpec with EncodeRequest {
```

Comment:
please run `fmt` on sbt shell and commit the changes

Reply:
Oops. Thanks for the feedback. Done